app/helpers/interruption_screen_helper.rb

```diff
@@ -10,7 +10,7 @@ def display_informative_text(informative_text, lettings_log)
               else
                 lettings_log.public_send(argument["key"])
               end
-      translation_params[argument["i18n_template"].to_sym] = value
+      translation_params[argument["i18n_template"].to_sym] = argument["prefix"] == "£" ? "£#{ActionController::Base.helpers.number_to_currency(value, delimiter: ',', format: '%n')}" : value
```

Comment:
these allow currency to display correctly even when not from a field in a log

Reply:
I think there's a bit of indirection here because the first condition isn't the primary path. Shall we talk this through to see if we can come up with something a bit more cleaner?